java/api/Driver.java

```diff
@@ -70,26 +71,14 @@ public interface Driver extends AutoCloseable {
      */
     void close();
 
-    /**
-     * Returns the logged-in user for the connection. Only for TypeDB Cloud.
-     *
-     * <h3>Examples</h3>
-     * <pre>
-     * driver.user();
-     * </pre>
-     */
-//    @CheckReturnValue
-//    User user();
```

Comment:
Removed as it's a duplication and a change from Rust. Can return, but it's generally `users().getCurrentUser()` or something like that (now it's `getCurrentUsername()`, but I was waiting for @lolski to implement it for Rust to propagate. Can leave it for you now, I guess).